packages/dds/tree/src/feature-libraries/sequence-field/types.ts

```diff
@@ -171,7 +128,7 @@ export interface Rename {
 	readonly idOverride: CellId;
```

Comment:
On line 123: "Occurs when an Attach is composed with a Detach."
Actually we could make that a little more specific: "Occurs when an Attach is composed with a Detach that leaves the cell with a different ID than it had before the Attach."